src/modules/user/user.service.ts

```diff
@@ -284,6 +296,38 @@ export class UserService {
     }
   }
 
+  async getWorkInfo(user: UserDto): Promise<WorkHistory[]> {
+    try {
+      const Workhistories = await this.workHistoryRepository
```

Comment:
no capital letters for the constants, where have you found it? this week you are 3d or 4th who follow this approach. if you go with upper case - everywhere uppercase (but senior developers will hate you:) ) if go with lover case - only lowercase

Reply:
yeah...that's awful, agree! Fixed